clients/nasaClient.ts

```diff
@@ -0,0 +1,18 @@
+import dotenv from "dotenv"
+import axios from "axios"
+import Rover from "../models/roverData"
+
+dotenv.config({path: './.env'})
+
+const apiKey: string | undefined = process.env.NASA_API_KEY
+export default class nasaClient {
+
+    static getRoverList() {
```

Comment:
Since we're using typescript, we'll want to add a return type here to explain what this function is returning